packages/gatsby-transformer-csv/src/__tests__/gatsby-node.ts

```diff
@@ -1,25 +1,33 @@
-const { parse: json2csv } = require(`json2csv`)
-const os = require(`os`)
-const { cloneDeep } = require(`lodash`)
+import { parse as json2csv } from "json2csv"
+import os from "os"
+import { cloneDeep } from "lodash"
 
-const { onCreateNode } = require(`../gatsby-node`)
-const { typeNameFromDir, typeNameFromFile } = require(`../index`)
+import { onCreateNode } from "../gatsby-node"
+import { typeNameFromDir, typeNameFromFile } from "../index"
+// import { IGatsbyNode } from "gatsby/internal"
+type IGatsbyNode = any
```

Comment:
I want to highlight this, in case it's missed. I get errors in the full CCI process if I attempt that import for real. I don't see any other modules importing IGatsbyNode, so I suspect it's not meant to be done,